moz_sql_parser/formatting.py

```diff
@@ -128,7 +135,7 @@ def op(self, json):
             return self._on(json)
 
         if len(json) > 1:
-            raise Exception('Operators should have only one key!')
+            self.dispatch(json)
```

Comment:
be sure this raises an exception, otherwise it will go into a recursive loop

Reply:
Changes completed. Thanks for the review.